core-services/prow/02_config/amq-streams-qe/strimzi-kafka-operator/_pluginconfig.yaml

```
@@ -0,0 +1,4 @@
+plugins:
+  amq-streams-qe/strimzi-kafka-operator:
+    plugins:
+      - trigger
```

```suggestion
    - trigger
```
This just failed due to syntax I believe removing these two spaces should fix.